### DISCUSSION TITLE
Possible Deprecation / Removal of D3D Backend

### DISCUSSION THREAD

**Discussion by pragmatrix** [rust-skia/rust-skia#741] [General]
The Skia team is planning to remove the D3D backend:

https://groups.google.com/g/skia-discuss/c/WY7yzRjGGFA?pli=1

**Comment by tronical** [comment]
Interesting. What's the alternative here? Use Dawn on Windows and everywhere else? (first on Skia level, I guess Rust bindings for Dawn is another question)
